src/aiko_services/main/pipeline.py

```diff
@@ -1077,7 +1084,23 @@ def destroy_stream(self, stream_id, graceful=False, use_thread_local=True):
                         element_name, stream, stream_event, diagnostic,
                         in_destroy_stream=True))
                     if stream.state == StreamState.ERROR:
-                        break
```

Comment:
This break prevents `stop_stream` being called on downstream elements if an upstream `stop_stream` raises